core/misc/dialog.js

```diff
@@ -75,7 +75,35 @@ Backdrop.dialog = function (element, options) {
         })
         .dialog('widget').css('position', 'fixed');
       Backdrop.optimizedResize.add(resetPosition, 'dialogResize.' + dialogId);
-      resetPosition();
+
+      // Because of a bug in Chromium, we must wait until the ajax-loaded
+      // jquery.ui.dialog.css is applied to the content of the modal through the
+      // class .ui-dialog-content. 
```

Comment:
Can we reduce this comment a bit?

I'm not actually sure, if I'd call that a bug BTW. It's just "behavior".

Reply:
I've trimmed the documentation.